libevm/VMFactory.cpp

```diff
@@ -175,30 +175,32 @@ po::options_description vmProgramOptions(unsigned _lineLength)
 }
 
 
-std::unique_ptr<VMFace> VMFactory::create()
```

Comment:
Is this change also needed or is enough to change the loader not to return a `unique_ptr`?

Reply:
Yes, because now we need two kinds of pointers, one with default deleter, one with null deleter.